src/lib.rs

```diff
@@ -0,0 +1,389 @@
+//! `fm` is a simple non-backtracking fuzzy text matcher useful for matching multi-line patterns
+//! and text. At its most basic the wildcard operator (`...` by default) can be used in the
+//! following ways:
+//!
+//!   * If a line consists solely of `...` it means "match zero or more lines of text".
+//!   * If a line starts with `...` it means "match the remainder of the line only".
+//!   * If a line ends with `...`, it means "match the start of the line only".
+//!   * If a line starts and ends with `...` it means "match the inner part of the line".
```

Comment:
Can `...` be used at both the beginning and end of the line simultaneously?

Reply:
(I think the answer is yes, but in the above list it's not clear)

Reply:
I don't know how to be clearer than "starts and ends with `...`" :)

Reply:
You're right. I misread. Sorry.